### PR TITLE
Bug 1706103 : Dockerfile: set coreos org as canonical for release-3.3

### PR DESCRIPTION
**Pull request by hexfusion**
This was an oversight by me but it must be fixed before GA.

Fixes: https://bugzilla.redhat.com/show_bug.cgi?id=1706103
Requires: https://github.com/openshift/release/pull/3693

### CODE REVIEW

File: Dockerfile.openshift [hunk 1]
```diff
@@ -1,20 +1,18 @@
 FROM registry.svc.ci.openshift.org/openshift/release:golang-1.10 AS builder
 
-ENV GOPATH /go
+WORKDIR /go/src/github.com/coreos/etcd
 
-COPY . $GOPATH/src/go.etcd.io/etcd
+COPY . .
 
-RUN yum install -y git && \
-    cd $GOPATH/src/go.etcd.io/etcd && \
```
Comment (hexfusion) [inline Dockerfile.openshift]: git is already part of the image.

**Comment by smarterclayton** [comment]
If you have release ready we should merge this, then release.

**Comment by hexfusion** [comment]
@smarterclayton can you bless this, please I am coordinating with release.

**Pull request merged**